src/libs/actions/Report.js

```diff
@@ -1225,6 +1157,107 @@ Onyx.connect({
     callback: handleReportChanged,
 });
 
+/**
+ * Deletes a comment from the report, basically sets it as empty string
+ *
+ * @param {Number} reportID
+ * @param {Object} reportAction
+ */
+function deleteReportComment(reportID, reportAction) {
+    const sequenceNumber = reportAction.sequenceNumber;
+
+    const optimisticReportActions = {
+        [sequenceNumber]: {
+            pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
+        },
```

Comment:
**Online behavior:**

Action should immediately be deleted.

**Offline behavior:**

Action shows as pending with the strikethrough 

How will we get the online behavior to work?

Reply:
I think [this line](https://github.com/Expensify/App/blob/main/src/components/OfflineWithFeedback.js#L84) guarantees that since we are setting pendingAction as 'delete'!